lib/ansible/modules/cloud/azure/azure_rm_devtestlabarmtemplate_facts.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2018 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_devtestlabarmtemplate_facts
+version_added: "2.8"
+short_description: Get Azure Arm Template facts.
+description:
```

Comment:
arm to full name?

Reply:
updated to "Azure DevTest Lab ARM Template"